master/getting-started/docker/tutorials/advanced-policy.md

```diff
@@ -14,23 +14,16 @@ driver creates a profile object for that network.  The default policy applied
 by Calico when a new network is created allows communication between all 
 containers connected to that network, and no communication from other networks.
 
-Using the standard `calicoctl profile` commands it is possible to manage the
-feature-rich policy associated with a network.
+Using the standard `calicoctl create` with a `profile` resource, it is possible to 
+manage the feature-rich policy associated with a network.
 
 > Note that if you want to access the feature rich Calico policy, you must use
 > both the Calico Network _and_ Calico IPAM drivers together.  Using the Calico
 > IPAM driver ensures _all_ traffic from the container is routed via the host
-> vRouter and is subject to Calico policy. Using the default IPAM driver 
-> instructs the Calico network driver to route non-network traffic (i.e.
-> destinations outside the network CIDR) via the Docker gateway bridge, and in
-> this case may not be subjected to the policy configured on the host vRouter.
+> vRouter and is subject to Calico policy.
 
 > The profile that is created by the Calico network driver is given the same 
-> name as the Docker-generated network ID.  This can be cumbersome to work
-> with, so the `calicoctl` tool that is used to manage the network policy 
-> handles the mapping between network names and IDs.  For the most part,
-> the detail about network IDs can be ignored, but it is mentioned here as a
-> side note for advanced users and developers,
+> name as the Docker network name.
```

Comment:
Probably ok for now, but I wonder if we should re-spin this a little to be that a Calico profile (of the same name as the network) can be used to control policy of containers in a particular network.  You may create the profile in advance, but if it does not exist it will be created when you create the first endpoint - the default configuration being to no isolation between all containers within that network.